tests/lib/util/node-event-generator.js

```diff
@@ -89,7 +89,7 @@ describe("NodeEventGenerator", () => {
 
             emitter.emit = (selector, node) => emissions.push([selector, node]);
 
-            estraverse.traverse(ast, {
+            Traverser.traverse(ast, {
```

Comment:
Oops, I hadn't realized this was a test -- I thought this was in `lib/util/node-event-generator`.  :+1: